content/en/docs/collector/howto-custom-collector.md

```diff
@@ -0,0 +1,112 @@
+# HOW-TO: Build your own Collector instance
+
+If you are planning to build and debug your components, you are going to need your own Collector instance. That will allow you to launch and debug your Collector components directly within your favorite Golang IDE.
```

Comment:
say "OpenTelemetry Collector" at the first instance of "Collector"

Reply:
@svrnm, fixed on [7d3d01](https://github.com/open-telemetry/opentelemetry.io/pull/1277/commits/7d3d01bd356ff8c6ae72ffd52cc476f69108f7ca)